engine/combat/combat_controller.py

```diff
@@ -2,14 +2,14 @@
 from typing import Self
 
 from control import sos_ctrl
-from engine.combat.appraisals.valere.crescent_arc import CrescentArc
-from engine.combat.appraisals.zale.sunball import Sunball
+from engine.combat.appraisals.basic_attack import BasicAttack
+from engine.combat.appraisals.valere import CrescentArc
+from engine.combat.appraisals.zale import Sunball
```

Comment:
Could even go one further an put a `__init__.py` in `engine/combat/appraisals`. Might get messier in the long run though.

Reply:
I'll leave it for now - if it gets too crazy in the future i'll combine them, but they make sense apart at the moment.